### ISSUE TITLE
UX: Systemintegration not as a Plugin

### ISSUE THREAD

**Issue by BobbySlope** [hoobs-org/HOOBS#107]
My idea:
We need to rethink that is all connected to Apple Homekit
Homekit is no longer the only system integration HOOBS can now also Google Home and Alexa

We should preinstall google home and Alexa and make a separate Tab for System Integrations to the Main menu with an icon like this https://www.flaticon.com/free-icon/business-affiliate-network_66585#term=connect&page=1&position=6

there should Homekit, Google Home and Alexa be listed
Perhaps also in Future Releases, KNX, Homeassistant, Openhab etc.

**Comment by mkellsy** [comment]
But the base of it all uses HAP. HomeKit Accessory Protocol. Home Assistant and OpenHAB are competing projects.

**Comment by BobbySlope** [comment]
Yes but for a user it doesn’t matter what n which protocol it works. You can use Homebridge without beeing connected to homekit. It works also only with google home

**Comment by mkellsy** [comment]
The Alexa and Home plugins are very vary large. Having that built into the core, would not be favorable. The core already takes long enough. Adding Google Home and Alexa to that would increase the install and upgrade time by 6 fold.

**Comment by mkellsy** [comment]
Planed for a different project.
